compiler/il/symbol/OMRResolvedMethodSymbol.cpp

```diff
@@ -1821,14 +1821,23 @@ OMR::ResolvedMethodSymbol::insertStoresForDeadStackSlotsBeforeInducingOSR(TR::Co
    TR::TreeTop *prev = induceOSRTree->getPrevTreeTop();
    TR::TreeTop *next = induceOSRTree;
 
-   TR::OSRTransitionTarget pointType = TR::preAndPostExecutionOSR;
+   /*
+    * Any arguments used by this bytecode index should be
+    * kept live
+    */
+   TR_BitVector *deadSymRefs = osrMethodData->getLiveRangeInfo(byteCodeIndex);
```

Comment:
Is this change also related to the pre/post OSR distinction that ceased to exist because of argument stashing ? I mean if we need to do this unconditionally now, why was it not being done under pre OSR before ?